src/azure-cli/azure/cli/command_modules/resource/tests/latest/subscription_level_template.json

```diff
@@ -4,7 +4,7 @@
   "parameters": {
     "storageAccountName": {
       "type": "string",
-      "defaultValue": "armbuilddemo1801"
+      "defaultValue": "cliarmbuilddemo1804"
```

Comment:
Can we pass in this parameter by random value? If it is written as a fixed value, sometimes it will affect the test results of our live test

Reply:
I agree. I changed the value since storage account names have to be globally unique and this test repeatedly failed as a result.

Reply:
You may modify it by referring to this PR: https://github.com/Azure/azure-cli/pull/14962/files#diff-ef876d58099baf761fb6f20c0aa11991830c232a37b137875c103839524b684cR676